src/components/Person/Person.jsx

```diff
@@ -1 +1,14 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    { person.age ? <p className="Person__age">{`I am ${person.age}`}</p> : ''}
```

Comment:
no need to use ternary here
```suggestion
    {age && <p className="Person__age">I am {age}</p>}
```